lib/puppetserver/ca/action/list.rb

```diff
@@ -209,8 +210,14 @@ def separate_certs(all_certs)
           return requested, signed, revoked
         end
 
-        def get_all_certs(settings)
-          result = Puppetserver::Ca::CertificateAuthority.new(@logger, settings).get_certificate_statuses
+        def get_all_certs_or_csr(settings, queried_state = nil)
+          if queried_state
+            query = { :state => queried_state }
+          else
+            query = {}
+          end
```

Comment:
Interested in feedback on making this if/else into a single line through the use of a ternary operator.

Reply:
```rb
query = queried_state ? { state: queried_state } : {}
```
Seems fine, this might be a little more idiomatic? Not sure..
```rb
query = {}
query[:state] = queried_state if queried_state
```

Reply:
yeah I kinda like the latter one better, but don't feel super strongly.